spec/models/article_spec.rb

```diff
@@ -1,19 +1,31 @@
 RSpec.describe Article, type: :model do
-
   describe 'Database' do
     it { is_expected.to have_db_column(:title).of_type(:string) }
     it { is_expected.to have_db_column(:body).of_type(:text) }
     it { is_expected.to have_db_column(:author).of_type(:string) }
   end
 
   describe 'Associations' do
-    it { is_expected.to belong_to :category}  
+    it { is_expected.to belong_to :category }
   end
-  
+
   describe 'Factory' do
     it 'is expected to be valid' do
       expect(create(:article)).to be_valid
     end
   end
 
-end
```

Comment:
lots of new codes and methods. which haven't seen before.

Reply:
From CA video re file attachments.